examples/looker.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * This script will automaticly look at the closest entity.
+ * It checks for a near entity every tick.
+ */
+const mineflayer = require('mineflayer')
+
+if (process.argv.length < 4 || process.argv.length > 6) {
+  console.log('Usage : node echo.js <host> <port> [<name>] [<password>]')
+  process.exit(1)
+}
+
+const bot = mineflayer.createBot({
+  host: process.argv[2],
+  port: parseInt(process.argv[3]),
+  username: process.argv[4] ? process.argv[4] : 'looker',
+  password: process.argv[5]
+})
+
+bot.once('spawn', function () {
+  setInterval(() => {
+    var entity = nearestEntity()
```

Comment:
There is also a `var` which is pretty much unsafe. You should use `const` there.